vllm/model_executor/models/qwen2_5_vl.py

```diff
@@ -315,7 +324,7 @@ def forward(
                                             cu_seqlens_k=cu_seqlens,
                                             max_seqlen_q=max_seqlen,
                                             max_seqlen_k=max_seqlen,
-                                            dropout_p=0,
+                                            dropout_p=dropout_p,
```

Comment:
```suggestion
                                            dropout_p=0.0,
```

I think we can just do this?